hw_20/testBank.py

```diff
@@ -0,0 +1,105 @@
+"""Homework_20"""
+
+import unittest
+import logging
+from hw_20.bank import Deposit, Bank
+
+formatter = logging.Formatter('\n[%(asctime)s] - %(levelname)s - %(message)s')
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
You can use 1 logger across single application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

Reply:
done